src/SideBar.tsx

```diff
@@ -67,6 +81,9 @@ export const SideBar = ({
           onUpdateConversationTitle={onUpdateConversationTitle}
         />
       </div>
+
+      {/*todo - implement modal*/}
+      {isSearchHistoryOpen && searchableHistory && <></>}
```

Comment:
We will eventually render the modal when the user has clicked the button to update the boolean to true and the data is successfully fetched and set in state.